src/generators/minecraft-character-allay/MinecraftAllayCharacterGenerator.res

```diff
@@ -0,0 +1,1154 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "minecraft-allay-character"
+
+let name = "Minecraft Allay Character"
+
+let images: array<Generator.imageDef> = [{id: "Overlay", url: requireImage("OverlayAllay.png")}]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin1",
+    url: requireTexture("Skin1.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Skin2",
+    url: requireTexture("Skin2.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+]
+
+module TextureMap = {
+  type cuboid = {
+    top: Builder.rectangle,
+    bottom: Builder.rectangle,
+    front: Builder.rectangle,
+    right: Builder.rectangle,
+    left: Builder.rectangle,
+    back: Builder.rectangle,
+    topoverlay: Builder.rectangle,
+    bottomoverlay: Builder.rectangle,
+    frontoverlay: Builder.rectangle,
+    rightoverlay: Builder.rectangle,
+    leftoverlay: Builder.rectangle,
+    backoverlay: Builder.rectangle,

```

Comment:
Let's use [camelCase](https://en.wikipedia.org/wiki/Camel_case) for all variable and function names. E.g. `topoverlay` -> `topOverlay`.